### ISSUE TITLE
NoMethodError: undefined method `getutc' when performance_stats is enabled

### ISSUE THREAD

**Issue by sgara** [airbrake/airbrake-ruby#487]
Hi,
We just updated airbrake-ruby to 4.5.0 and started seeing errors like
```
NoMethodError: undefined method `getutc' for 237.488926526:Float
```
Partial stack trace: 
```
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/time_truncate.rb:12 in utc_truncate_minutes
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/request.rb:40 in to_h
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/hash_keyable.rb:34 in hash
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/performance_notifier.rb:57 in update_payload
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/performance_notifier.rb:37 in block in notify
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/performance_notifier.rb:36 in synchronize
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby/performance_notifier.rb:36 in notify
/gems/airbrake-ruby-4.5.0/lib/airbrake-ruby.rb:358 in notify_request
/gems/airbrake-9.3.0/lib/airbrake/rails/action_controller_notify_subscriber.rb:17 in block in call
/gems/airbrake-9.3.0/lib/airbrake/rails/action_controller_notify_subscriber.rb:16 in each
/gems/airbrake-9.3.0/lib/airbrake/rails/action_controller_notify_subscriber.rb:16 in call
/gems/activesupport-6.0.0.rc1/lib/active_support/notifications/fanout.rb:189 in finish
```

The error disappears when airbrake is disabled (development for instance) or when `performance_stats` is turned off

Running Rails 6.0.0-rc1 with ruby 2.6.2

**Comment by kyrylo** [comment]
Thanks for the report. It looks like this is a Rails bug/breaking change. I can see this issue where they changed `event.time` and `event.end` to return monotonic time instead of `Time`: https://github.com/rails/rails/issues/36145 & https://github.com/rails/rails/pull/35984. That change was released in Rails 6.0.0-rc1. That's why it breaks our notifier: we rely on the old behaviour.

That said, I also noticed that it was reverted in https://github.com/rails/rails/pull/36148/files. So current Rails `master` and `6.0.0-rc1` have different code. Later on Rails introduced `ActiveSupport::Notifications.monotonic_subscribe` that works exactly how it `ActiveSupport::Notifications.subscribe` works in `6.0.0-rc1` (we use this API and it fails for you).

TL;DR: I think we don't have to do anything but wait for the new Rails release. I tried to run Rails `master` to verify this but it won't allow me to install certain gems. If this continues with `6.0.0-rc1+`, then I'll take a look again, but I believe we're good.

**Comment by jamesst20** [comment]
I can confirm this has been fixed in Rails 6 RC2. The bug was present with RC1
